src/pages/_document.tsx

```diff
@@ -1,14 +1,43 @@
 import {Theme} from '@gravity-ui/page-constructor';
 import block from 'bem-cn-lite';
 import {Head, Html, Main, NextScript} from 'next/document';
+import Script from 'next/script';
+
+import {IS_PRODUCTION} from '../constants';
+
 const theme = Theme.Dark;
 const b = block('yc-root');
 
+const GA_ID = 'GTM-KHT6KD7';
```

Comment:
From my points of view `config.ts` or `constants.ts` is the better place for this

Reply:
done